project/src/main/webapp/index.css

```diff
@@ -7,7 +7,7 @@ html {
 
 .hero-image {
   background-image: linear-gradient(rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)),
-    url("https://as1.ftcdn.net/jpg/02/85/12/82/500_F_285128239_yUAIjW8lOpIeUaVfUFo3YerJN14fJaD7.jpg");
+    url("https://image.freepik.com/free-vector/crowd-people-walking-outdoors-night-market_74855-7109.jpg");
```

Comment:
use https://screenshot.googleplex.com/C6PuyjD5ffEKJ3o.png (same img just cropped and hosted by me aka snipit)

Reply:
Done